.github/workflows/build-ios.yml

```
@@ -25,11 +25,14 @@ on:
 jobs:
   build:
     if: github.repository == 'Expensify/react-native-live-markdown'
-    runs-on: macos-13
+    runs-on: macos-15
     concurrency:
       group: build-ios-${{ github.ref }}
       cancel-in-progress: true
     steps:
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: 16.1
```

Instead of relying on third-party actions, you can just use `xcode-select` command directly.

https://github.com/actions/runner-images/blob/main/images/macos/macos-15-Readme.md#xcode

```suggestion
      - run: sudo xcode-select -s /Applications/Xcode_16.1.app
```